content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -424,3 +427,17 @@ namespace.
 
 When each team has their own mesh, cross-mesh communication follows the
 concepts described in the [multiple meshes](#multiple-meshes) model.
+
+## Virtual machines and bare metal
+
+In same cases you might have to deploy some of the services on virtual machines or bare metal computers. The
+reasons for deployments on VMs/bare metal computers could be special hardware requirements, tighter isolation, tighter integration with the host's OS, legacy deployments.
+
+Having some of the services on VMs/bare metal, you may want to connect the services into a mesh and benefit from all the
+service mesh features: traffic management, security, observability, and others. You can deploy the Istio control plane(s)
```

Comment:
@rcaballeromx please note that the goal is to present Istio features in general, since this is a high-level statement. The features of Istio on general: traffic management, security, observability. The list of the Istio features is too detailed to present the reader in a high level description.

Reply:
The problem is that your list is not complete. This is also not the place to present Istio features in general. Sometimes, less is more. As @frankbu and @smawson stated, this doc is not about showcasing why folks should add VMs to their deployments.

Reply:
Note that we do not know if all the features of Istio work for VMs. So we cannot state that the users will benefit from all the Istio features when they integrate VMs with the mesh. However, we still want to provide users some motivation to integrate VMs with the mesh. So what should be stated here: "you will benefit from some of the features of Istio"? Or "you will benefit from most of the features"?

>  this doc is not about showcasing why folks should add VMs to their deployments

The reader could ask a reasonable question: what do I gain by integrating VMs with the mesh?